dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -308,10 +308,15 @@ ERROR_TRANSFORMING_CLASS_CWWJP0055E.useraction=The class cache might be corrupte
 
 #-------------------------------------------------------------------------------
 # 0=property name, 1=property value
-EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured in the server.xml file contains an empty name or empty value: name = [{0}], value = [{1}]. The property will be ignored.
+EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property, configured in the server.xml file, contains an empty name or empty value: name = [{0}], value = [{1}]. The property will be ignored.
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.explanation=The name and value attributes for a JPA default property must not be empty.
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.useraction=Change the server configuration to provide a non-empty name and value for the property or remove the property.
 
+# 0=property name, 1=property value
+JDBC_PROP_NAME_CWWJP0057W=CWWJP0057W: A JPA default property, configured in the server.xml file, is intended for use in Java SE environments only: name = [{0}], value = [{1}]. The property will be ignored.
```

Comment:
misplaced "only"

A JPA default property that is intended for use only in Java SE environments is configured in the server.xml file : name = [{0}], value = [{1}]. This property is ignored.